platform/android/gradle/dependencies.gradle

```diff
@@ -8,24 +8,26 @@ ext {
     ]
 
     versions = [
-            mapboxServices: '3.0.0-beta.2',
+            mapboxServices : '3.0.0-beta.2',
             mapboxTelemetry: '3.0.0-beta.1',
-            supportLib    : '25.4.0',
-            espresso      : '3.0.1',
-            testRunner    : '1.0.1',
-            leakCanary    : '1.5.1',
-            lost          : '3.0.4',
-            junit         : '4.12',
-            mockito       : '2.10.0',
-            robolectric   : '3.5.1',
-            timber        : '4.5.1',
-            okhttp        : '3.9.1'
+            mapboxGestures : '0.1.0-20180219.142215-11',
```

Comment:
Noting that we rely on a particular `SNAPSHOT` of the gestures library at the moment. This will need to change when the final release of 6.0.0 comes, synced with the first stable release of the gestures library.